src/components/shared/Entity/Actions/SaveButton.tsx

```diff
@@ -1,24 +1,16 @@
-import { useEditorStore_id } from 'components/editor/Store/hooks';
 import EntityCreateSave from 'components/shared/Entity/Actions/Save';
-import useEntityWorkflowHelpers from 'components/shared/Entity/hooks/useEntityWorkflowHelpers';
-import { useFormStateStore_status } from 'stores/FormState/hooks';
-import { FormStatus } from 'stores/FormState/types';
-import useEntityWorkflowHydrated from '../hooks/useEntityWorkflowHydrated';
+import useEntitySaveHelpers from '../hooks/useEntitySaveHelpers';
 import { EntitySaveButtonProps } from './types';
 
 function EntitySaveButton({ logEvent, disabled }: EntitySaveButtonProps) {
-    const { callFailed } = useEntityWorkflowHelpers();
-    const formsHydrated = useEntityWorkflowHydrated();
-
-    const draftId = useEditorStore_id();
-
-    const formStatus = useFormStateStore_status();
+    const { onFailure, buttonDisabled, formSaving } =
+        useEntitySaveHelpers(disabled);
```

Comment:
Started making the stuff related to save shared. That path is not being gone down anymore... I think... so still leaving this hook in.